tests/ethpm/packaging-examples/owned/1.0.0-pretty.json

```diff
@@ -0,0 +1,21 @@
+{
+  "manifest_version": "2",
+  "version": "1.0.0",
+  "package_name": "owned",
+  "meta": {
+    "license": "MIT",
+    "authors": [
+      "Piper Merriam <pipermerriam@gmail.com>"
```

Comment:
Side note: you guys should probably replace all of these with `author@example.com` or some other acceptable lorem-ipsum style address.